Assets/Scripts/ChangeFrame.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Security.Cryptography;
+using UnityEngine;
+
+public class ChangeFrame : MonoBehaviour
+{
+    public Action onIncreaseFrame;
+    public Action onDecreaseFrame;
+
+    public void IncreaseFrame()
+    {
+        onIncreaseFrame?.Invoke();
```

Comment:
find a way to remove the null check

Reply:
Why ?

Reply:
null checks are bad code smell if you have learned your lessons well